app/services/geocoder_service.rb

```diff
@@ -5,6 +5,10 @@ def self.geocode(obj:, force: false)
     if result.present?
       obj.latitude = result.latitude
       obj.longitude = result.longitude
+
+      region_code = COUNTY_TO_REGION_CODE[result.district]
+      obj.region_code = region_code unless region_code.blank?
```

Comment:
so after geo code....

can still be empty... but don't set it to be nil?

Reply:
Hmm, the intention here is to only set it to something if region_code is not empty string or blank; `''.blank?` is `true`:

```ruby
[5] pry(main)> !''.blank?
=> false
[6] pry(main)> !nil.blank?
=> false
```

Makes sense?

Reply:
oh you want this

 obj.region_code = region_code if region_code.present?

Reply:
Yes 👍